### PR TITLE
Update egg-insurgency--sandstorm.json

### PR DESCRIPTION
**Pull request by xardoniak**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [y ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [n ] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?: not sure here - new to github

### Changes to an existing Egg:
-Fixed Max Players variable
-Added GSL Token
-Added GS Token

1. [y ] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [y ] Have you tested your Egg changes?

### CODE REVIEW

**Comment by Software-Noob** [comment]
The list is marked as done or not done by typing X inside of the brackets.

You're not the original author, so I think the removal of the author's email is kind of bad. Your egg can't be imported at this moment due to the "file_denylist" line in the JSON, this should be resolved in the next panel release but until then you could remove that line.

I would pre-define default for RCON port together with making both map cycle and RCON password variables nullable as they're not required. RCON password being empty disables RCON.

Other than that, everything is working as it should. You could also change the script container to debian:buster-slim while you're at it.

**Comment by xardoniak** [comment]
Oh whoops I forgot the email was updated! Now that you've made the point, I'm in agreeance!

I'll have a look at making those changes

File: steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v1",
+        "update_url": null
     },
-    "exported_at": "2020-06-27T05:17:21-04:00",
+    "exported_at": "2021-02-16T10:20:13+11:00",
     "name": "Insurgency: Sandstorm",
-    "author": "brycea@terrahost.cloud",
+    "author": "playstrikemc@outlook.com",
```
Comment (parkervcp) [inline steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json:9]: Why was this changed?

File: steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
     "description": "Insurgency: Sandstorm is a team-based, tactical FPS based on lethal close quarters combat and objective-oriented multiplayer gameplay. Experience the intensity of modern combat where skill is rewarded, and teamwork wins the fight.",
-    "image": "quay.io\/pterodactyl\/core:source",
-    "startup": "\/home\/container\/Insurgency\/Binaries\/Linux\/InsurgencyServer-Linux-Shipping {{MAP_NAME}}?Scenario={{SCENARIO}}?MaxPlayers={{MAX_PLAYERS}} -Port={{SERVER_PORT}} -QueryPort={{QUERY_PORT}} -log -hostname=\"{{HOSTNAME}}\"",
+    "features": null,
+    "images": [
+        "quay.io\/pterodactyl\/core:source"
+    ],
+    "file_denylist": "",
```
Comment (parkervcp) [inline steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json:15]: This will cause it to fail to import currently.

File: steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json (hunk 1 above)
Comment (xardoniak) [reply steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json:9]: Hi Parker,
I believe playstrikemc was the original author of the egg. I made some tweaks to a copy that another user had already tweaked and they must have updated the email address :)

just setting it back to the original author

Comment (parkervcp) [reply steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json:9]: bryce is the only one with their name on this file and it hasn't changed for 8 months since they uploaded it. Please change it back.

**Comment by parkervcp** [comment]
None of the requested changes have been made from when I asked for them I will close this out. If the change is made we can re-open and discuss merging/
